libs/utils/src/polls.ts

```diff
@@ -70,7 +72,7 @@ export function getPollTransitionsFromState(
     case 'polls_open':
       return ['close_polls', 'pause_polls'];
     case 'polls_paused':
-      return ['open_polls', 'close_polls'];
+      return ['unpause_polls', 'close_polls'];
```

Comment:
Making the language explicit in the UI helped identify this actual error :)

Reply:
Is there a test that covers this?

Reply:
https://github.com/votingworks/vxsuite/pull/2764/commits/63cc1eddb097e56a4bd22babcfe617ec2b13418e#diff-797af4cc4e0b6e4d7b559398a2027267f42f8828ca9f5afd0dae710e73c61315R104

Reply:
These sorts of tests where the assertions just basically rehash the behavior of the utility seem useless to me.

Reply:
But now that the language is distinct, this is covered in a few places in the frontends.